packages/flutter/test/scheduler/binding_test.dart

```diff
@@ -14,6 +14,16 @@ void main() {
     expect(SchedulerBinding.instance.platformDispatcher.onBeginFrame, isNotNull);
   });
 
+  test('Ticker.forceFrames requests forced frames', () async {
+    final Ticker t = Ticker((_) {});
+    t.forceFrames = true;
+    final TickerFuture f = t.start();
+    // A forced frame should be scheduled even if frames are otherwise disabled.
+    expect(SchedulerBinding.instance.hasScheduledFrame, isTrue);
+    t.stop();
```

Comment:
Should this be done in an addTearDown so it's still stopped even if the previous expectation fails?